README.md

```diff
@@ -1,62 +1,57 @@
-Kute App
-===================
+# What is Kute?
+A Commute App for Sri Lanka
 
+![cover](https://user-images.githubusercontent.com/17242746/34432772-7414905c-eca0-11e7-874e-c38a2c96671f.jpg)
 
-**Kute** is an app that makes navigation and transportation easier and more efficient for people travelling to the same destination.People can easily navigate and identify public transport services around them without having to look through the schedules and timings.Travellers can share their location and tackle the problems of ever-changing schedules together.
+>People travel to the same destination on a regular basis for various reasons using _public and private_ transport. In **Kute**, the users (also known as travellers) themselves are allowed to share their current location, so that the other travellers can see the current location of the vehicle. A common type of vehicle used for public transport in Sri Lanka is the passenger train. Kute is currently implemented for trains. A registered user can update the current location of a train when travelling by sharing the current location. Another user can view location of a selected train and the expected time a particular train arrives at the nearest station using the location data updated by another user.
 
-![enter image description here](http://res.cloudinary.com/raghavp/image/upload/v1513531690/kute1_bhhvhj.png)
+# Features
+- Makes navigation and transportation easier.
+- People can navigate & identify public transport services.
+- Travellers can share their location.
 
-![enter image description here](http://res.cloudinary.com/raghavp/image/upload/v1513531691/kute2_vtvneu.png)
+![ss](https://user-images.githubusercontent.com/17242746/34433097-c0e5f16c-eca2-11e7-922e-086a2a7c0e91.png)
 
-----------
+# Setting up the Project
+- Fork the repository by clicking on the Fork icon at the top right corner of the repository page.
+- Clone the repository on to your local machine by running the following commands on git:
+- git clone git clone `https://github.com/[YOUR-USERNAME]/kute.git` Refer Forking and Cloning in -
```

Comment:
There's an additional "git clone" phrase. Also, the ending is not good. Please have a look.